src/sql/workbench/services/query/common/queryModelService.ts

```diff
@@ -409,6 +416,25 @@ export class QueryModelService implements IQueryModelService {
 		}
 	}
 
+	public async notifyConnectionUriChanged(newUri: string, oldUri: string): Promise<void> {
```

Comment:
This should also be `changeConnectionUri` - again since it's actually updating its own mapping in addition to notifying others to update themselves.